_translator-files/po/nl/Tips-Tricks-More.po

```diff
@@ -346,3 +323,18 @@ msgid ""
 "<?php loadserverstat ( \"stat1.dat\" ); ?>\n"
 "~~~\n"
 msgstr ""
```

Comment:
Is there no way to mark a block as "no translate required"?

Reply:
I believe we had this discussion when po4a was introduced and if I recall correct it is not possible.
Current stand is that when you don't add it to the nl text (which I tried in the beginning) it no longer counts for the amount of translation done and often (as these blocks can be large) will get the file below the threshold to get the translation visible.

Reply:
@henkdegroot is correct. It's just ctrl+b in Poedit to copy the source segment over.